src/theme/css/variables.css

```diff
@@ -1,701 +1,22 @@
-    --quote-border: #fe27c3;
-
-    --table-border-color: #91146f;
-    --table-header-bg: #2d0221;
-    --table-alternate-bg: #2d0221;
-
-    --searchbar-border-color: #fe27c3;
-    --searchbar-bg: #170312;
-    --searchbar-fg: #f76ed1;
-    --searchbar-shadow-color: #fe275b;
-    --searchresults-header-fg: #f76ed1;
-    --searchresults-border-color: #fe275b;
-    --searchresults-li-bg: #170312;
-    --search-mark-bg: #f76ed1;
-
-    --chapter-hover-bg: #170312
-}
-
-.kneon {
-    text-shadow: 0 0 10px #170312;
-}
-
-.kneon ::-webkit-scrollbar {
-    width: 15px;
-    height: 12px
-}
-
-.kneon code {
-    background: #35112b;
-}
-
-.kneon blockquote {
-    border-radius: 10px;
-}
-/* Kinda Neon */
-
-/* Shiro's Theme */
-.shiro {
-    --bg: hsl(0, 0%, 0%);
-    --fg: #c5c5c5;
-
-    --sidebar-bg: hsl(0, 0%, 0%);
-    --sidebar-fg: #c5c5c5;
-    --sidebar-non-existant: #5c6773;
-    --sidebar-active: #2e91fb;
-    --sidebar-spacer: #2e91fb;
-
-    --scrollbar: var(--sidebar-fg);
-
-    --icons: #737480;
-    --icons-hover: #2e91fb;
-
-    --links: #a689fc;
-
-    --inline-code-color: #2e91fb;
-
-    --theme-popup-bg: hsl(0, 0%, 0%);
-    --theme-popup-border: #2e91fb;
-    --theme-hover: #2e91fb;
-
-    --quote-bg: hsl(0, 0%, 0%);
-    --quote-border: #2e91fb;
-
-    --table-border-color: hsl(210, 25%, 13%);
-    --table-header-bg: hsl(210, 25%, 28%);
-    --table-alternate-bg: hsl(210, 25%, 11%);
-
-    --searchbar-border-color: #2e91fb;
-    --searchbar-bg: hsl(0, 0%, 0%);
-    --searchbar-fg: #2e91fb;
-    --searchbar-shadow-color: #d4c89f;
-    --searchresults-header-fg: #666;
-    --searchresults-border-color: #888;
-    --searchresults-li-bg: #252932;
-    --search-mark-bg: #2e91fb;
-    
-    --chapter-hover-bg: var(--theme-hover);
-}
-/* Shiro's Theme */
-
-/* Terminal */
-.terminal {
-    --bg: #0c0c0c;
-    --fg: #149414;
-
-    --sidebar-bg: #090808;
-    --sidebar-fg: #9ccc9c;
-    --sidebar-non-existant: #2b5329;
-    --sidebar-active: #649568;
-    --sidebar-spacer: #23242a;
-
-    --scrollbar: #2b5329;
-
-    --icons: #fff;
-    --icons-hover: #2b5329;
-
-    --links: #9ccc9c;
-
-    --inline-code-color: #c5c8c6;
+/* Main */
+@import './wiki-themes/root.css';
+@import './wiki-themes/breadcrumbs.css';
 
-    --theme-popup-bg: #0c0c0c;
-    --theme-popup-border: #0e6b0e;
-    --theme-hover: #2b5329;
-
-    --quote-bg: #0c0c0c;
-    --quote-border: #0e6b0e;
-
-    --table-border-color: #2b5329;
-    --table-header-bg: #2b5329;
-    --table-alternate-bg: #0e1c0d;
-
-    --searchbar-border-color: #9ccc9c;
-    --searchbar-bg: #000;
-    --searchbar-fg: #9ccc9c;
-    --searchbar-shadow-color: #9ccc9c;
-    --searchresults-header-fg: #9ccc9c;
-    --searchresults-border-color: #649568;
-    --searchresults-li-bg: #2b5329;
-    --search-mark-bg: #2b5329;
-
-    --chapter-hover-bg: var(--theme-hover);
-}
-
-.terminal a:not(.header) {
-    text-shadow: 0 0 10px#00ff05;
-}
-/* Terminal */
-
-/* High Contrast */
-.highcon {
```

Comment:
I said no to removing the `High Contrast` theme. Also, put it at the bottom of the theme list.